EpikCord.py/components.py

```diff
@@ -1,5 +1,5 @@
-from typing import Optional
-from .exceptions import InvalidArgumentType, CustomIdIsTooBig, TooManySelectMenuOptions, InvalidComponentStyle, TooManyComponents
+from typing import Optional, Union, List
+from .exceptions import InvalidArgumentType, CustomIdIsTooBig, TooManySelectMenuOptions, InvalidComponentStyle, TooManyComponents, LabelIsTooBig, MissingCustomId
```

Comment:
I already have this in one of my commits that haven't been pushed yet

Reply:
lol